<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPreferences</class>
 <widget class="QDialog" name="UserPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Préférences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Chemins:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fichier de sauvegarde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_file_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_search_but">
       <property name="text">
        <string>Choisir...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Langage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_language_box"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_search_but</sender>
   <signal>clicked()</signal>
   <receiver>UserPreferences</receiver>
   <slot>setFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFilePath()</slot>
 </slots>
</ui>
